src/main/java/Duke.java

```diff
@@ -6,5 +7,80 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Task[] userInput = new Task[100];
+        int inputCount = 0;
+        while(true) {
+            String line;
+            Scanner input = new Scanner(System.in);
+            line = input.nextLine();
+            if (line.equals("list")){
```

Comment:
Consider adding a space between the if statement and the braces as per k&r coding standard